comm/src/test/scala/coop/rchain/comm/connect/ClearConnectionsSpec.scala

```diff
@@ -97,45 +97,68 @@ class ClearConnectionsSpec
       it("should put the peers that responded to heartbeat to the end of the list") {
         // given
         implicit val connections = mkConnections(peer("A"), peer("B"), peer("C"), peer("D"))
-        implicit val max         = conf(maxNumOfConnections = 5, numOfConnectionsPinged = 3)
+        implicit val rpconf      = conf(maxNumOfConnections = 5, numOfConnectionsPinged = 3)
         transport.setResponses({
-          case p if (p == peer("A")) => alwaysFail
-          case _                     => alwaysSuccess
+          case p if p == peer("A") => alwaysFail
+          case _                   => alwaysSuccess
         })
         // when
         Connect.clearConnections[Id]
-        // tehn
-        connections.read.size shouldBe (3)
-        connections.read shouldEqual (List(peer("D"), peer("B"), peer("C")))
+        // then
+        connections.read.size shouldBe 3
+        connections.read shouldEqual List(peer("D"), peer("B"), peer("C"))
       }
 
       it("should report number of connections that were removed") {
         // given
         implicit val connections = mkConnections(peer("A"), peer("B"), peer("C"), peer("D"))
-        implicit val max         = conf(maxNumOfConnections = 5, numOfConnectionsPinged = 3)
+        implicit val rpconf      = conf(maxNumOfConnections = 5, numOfConnectionsPinged = 3)
         transport.setResponses({
-          case p if (p == peer("A")) => alwaysFail
-          case _                     => alwaysSuccess
+          case p if p == peer("A") => alwaysFail
+          case _                   => alwaysSuccess
         })
         // when
         val cleared = Connect.clearConnections[Id]
-        // tehn
-        cleared shouldBe (1)
+        // then
+        cleared shouldBe 1
+      }
+    }
+
+    describe("if dynamic local host changes") {
+      it("should disconnect from all peers and clear connections") {
```

Comment:
This should be in its own test `ResestConnectionsSpec` as "reset" has nothing to do with "dynami host change". We *know* we call it when host will change dynamically, but that is not implied in comm layer nor in tests. Description does not match the "//given". I would just move it to its own spec and defined that this is feature that resets the connections.